openpype/tools/project_manager/project_manager/widgets.py

```diff
@@ -141,16 +144,25 @@ def __init__(self, parent=None, dbcon=None):
         inputs_widget = QtWidgets.QWidget(self)
         project_name_input = QtWidgets.QLineEdit(inputs_widget)
         project_code_input = QtWidgets.QLineEdit(inputs_widget)
+        project_width_input = QtWidgets.QLineEdit(inputs_widget)
+        project_height_input = QtWidgets.QLineEdit(inputs_widget)
```

Comment:
and this also should be Int value input

Reply:
I think I have covered incorrect QLineEdit inputs with the `self._validate_number` method. Do you mean I need to use something more native like `QIntValidator`? Of use something else than QLineEdit for the input?

Reply:
In synchronicity this is duplicate comment ;) https://github.com/ynput/OpenPype/pull/5075#discussion_r1214146492